TeamViewerADConnector/Internal/Configuration.ps1

```diff
@@ -48,4 +49,8 @@ function Confirm-Configuration($config) {
     if ($config.UseSsoCustomerId -And [string]::IsNullOrWhiteSpace($config.SsoCustomerId)) {
         Throw "The parameter 'SsoCustomerId' cannot be empty if 'UseSsoCustomerId' is configured."
     }
+	$parseGuid = [System.Guid]::empty
```

Comment:
The check can be simplified, e.g. like this:
```suggestion
![string]::IsNullOrEmpty($meetingKey) -And [guid]$meetingKey | Out-Null
```

Also: tabs vs. spaces. I suggest using vscode + Powershell Extension to format the files

Reply:
That definitely feels less clunky. (Though I did add a comment just to clear why/what is happening.)

Reply:
Regarding the tabs: That one slipped through. There should not be any left now.